build.sbt

```diff
@@ -31,6 +31,8 @@ lazy val fable = (project in file("."))
       findJar((fullClasspath in Compile).value, "kafka-clients") -> url(
         "https://kafka.apache.org/21/javadoc/")
     ),
+    maxErrors := 5,
```

Comment:
I think this is generally better to configure per-user, ie in `~/.sbt/1.0/global.sbt` vs making it part of the project configuration.